static/app/views/admin/options.tsx

```diff
@@ -62,6 +66,34 @@ const HIGH_THROUGHPUT_RATE_OPTION = {
   step: 0.0001,
 };
 
+const autoRegisterOptionDefinitions: Field[] = [
+  ...autoRegisterDetectorOptions.flatMap(option => [
+    {
+      key: `performance.issues.${option.optionRoot}.problem-creation`,
+      label: t('Problem Creation Rate'),
+      help: tct(
+        'Controls the overall rate at which performance problems are detected by the [name] detector.',
+        {
+          name: option.detectorName,
+        }
+      ),
+      ...HIGH_THROUGHPUT_RATE_OPTION,
+    },
+    ...autoRegisterCohorts.map(cohort => ({
+      key: `performance.issues.${option.optionRoot}.${cohort}-rollout`,
+      label: tct('[cohort] Rate', {cohort: cohort.toUpperCase()}),
+      help: tct(
+        'Controls the rate at which performance problems are detected by the [name] detector for [cohort] organizations',
```

Comment:
Oh, I've been meaning to update this string, since it's kinda wrong, now's a good time?
```suggestion
        'Controls the percentage of [cohort] organizations that are eligible for [name] detector',
```